src/style.css

```diff
@@ -1 +1,28 @@
-/* add styles here */
+@import 'styles/stars1.css';
+@import 'styles/stars2.css';
+@import 'styles/stars3.css';
+@import 'styles/stars4.css';
+@import 'styles/stars5.css';
```

Comment:
The use of `@import` statements is a good practice for organizing styles. However, the task requires each BEM block to have its own separate CSS file. Ensure that each BEM block is independently managed and consider restructuring the imports to reflect this requirement.